runtime/altair/src/lib.rs

```diff
@@ -1501,6 +1604,101 @@ pub type Executive = frame_executive::Executive<
 	UpgradeAltair1027,
 >;
 
+mod session_key_migration {
+	use frame_support::{pallet_prelude::*, storage_alias};
+
+	use super::*;
+	pub struct ExtendSessionKeys;
+
+	impl_opaque_keys! {
+		pub struct OldSessionKeys {
+			pub aura: Aura,
+		}
+	}
+
+	#[storage_alias]
+	type QueuedKeys<T: pallet_session::Config> = StorageValue<
+		pallet_session::Pallet<T>,
+		// hex_literal::hex!("cec5070d609dd3497f72bde07fc96ba0"),
```

Comment:
I am not sure I get this comment; is this what the storage value prefix encodes to or is this the only value we expect to exist when running the migration?

Reply:
Sorry, this hex literal comment should be removed. It was part of my struggle to get this migration running.